tests/test_scripts/junit_tests_foundation.sh

```diff
@@ -16,5 +16,5 @@ then
 fi
 echo "TEST_DIR: $TEST_DIR"
 
-mvn -Dtest="org.xtreemfs.foundation.**, org.xtreemfs.test.foundation.**" -DfailIfNoTests=false --settings $XTREEMFS/java/settings.xml --activate-profiles xtreemfs-dev --file $XTREEMFS/java/pom.xml test 2>&1 | tee "${TEST_DIR}/log/junit.log"
+mvn -Dtest="org.xtreemfs.foundation.**, org.xtreemfs.foundation.flease.**" -DfailIfNoTests=false --settings $XTREEMFS/java/settings.xml --activate-profiles xtreemfs-dev --file $XTREEMFS/java/pom.xml test 2>&1 | tee "${TEST_DIR}/log/junit.log"
```

Comment:
I think the pattern `org.xtreemfs.foundation.**` covers all subdirectories, so the additional `.flease` should not be necessary.